### ISSUE TITLE
Build on Intel Mac

### ISSUE THREAD

**Issue by softee-p** [kismetwireless/lorcon#15]
Hello,

I am currently trying to make pylorcon2 work by building lorcon from source on a Macbook Air intel based.

I was trying to get airdrop-ng working on said macbook.

Instructions on the airdrop README lead to errors with make lorcon.

Before i spend more time with this, is it even possible to use the end result with that wireless card? It's the BCM4355 i believe.

**Comment by kismetwireless** [comment]
I don't think you can inject packets with the macos drivers, unfortunately.

Your best chance afaik would be to get a VM running and try using a USB card in linux that way via usb passthrough (not ethernet/nic passthrough, raw usb). It *might* work, usb passthrough can be a bit dodgy sometimes.

------- Original Message -------

**Comment by softee-p** [comment]
Yes you are right that method works. But that's not what i'm after.

I want to use the AP features of airdrop-ng. I'm asking here because lorcon/pylorcon2 are dependencies that i'm having trouble installing on macOS.

Sorry for the confusion.

Is there a way around this?

**Comment by kismetwireless** [comment]
As far as I know it is not possible to inject packets on macos, because apple does not provide that functionality and you can't load custom drivers for the airport cards. I don't think it's possible to do what you want to do under macos.

------- Original Message -------

**Comment by softee-p** [comment]
I would need custom drivers for the AP because airdrop operates it through that if i understood? 

Sounds like i have to keep using Linux for airdrop.

Thank you for clarifying.
